moonshot/integrations/cli/redteam/session.py

```diff
@@ -407,7 +719,6 @@ def delete_session(args) -> None:
     description="Runs automated red teaming in the current session.",
     epilog=(
         'Example:\n run_attack_module sample_attack_module "this is my prompt" -s "test system prompt" '
```

Comment:
so now we are no longer getting inputs from user on the context, prompt, and the metric? @imda-kelvinkok

Reply:
will add back the metric. context strategy and prompt template will no longer be passed in from the command but retrieved from the session